invenio_rdm_records/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+from flask_security.confirmable import confirm_user
+from flask_security.utils import hash_password
+from invenio_accounts.proxies import current_datastore
+from invenio_db import db
+from invenio_users_resources.services.users.tasks import reindex_user
+
+
+def get_or_create_user(email):
+    """Get or create a user."""
+    user = current_datastore.get_user(email)
+    if not user:
+        with db.session.begin_nested():
+            user = current_datastore.create_user(
```

Comment:
User does not have a `username` set. I would suggest to add one so it does not look like `user@demo.org <null>` in communities->members->invite